internal/fileutils/fileutils.go

```diff
@@ -76,53 +74,22 @@ func ReplaceAll(filename, find string, replace string, include includeFunc) erro
 		return nil
 	}
 
-	findBytes := []byte(find)
-	replaceBytes := []byte(replace)
-	replaceBytesLen := len(replaceBytes)
-
-	// Check if the file is a binary file. If so, the search and replace byte
-	// arrays must be of equal length (replacement being NUL-padded as necessary).
-	var replaceRegex *regexp.Regexp
-	quoteEscapeFind := regexp.QuoteMeta(find)
-
-	// Ensure we replace both types of backslashes on Windows
-	if runtime.GOOS == "windows" {
-		quoteEscapeFind = strings.ReplaceAll(quoteEscapeFind, `\\`, `(\\|\\\\)`)
-	}
-	if IsBinary(fileBytes) {
-		//logging.Debug("Assuming file '%s' is a binary file", filename)
-
-		regexExpandBytes := []byte("${1}")
-		// Must account for the expand characters (ie. '${1}') in the
-		// replacement bytes in order for the binary paddding to be correct
-		replaceBytes = append(replaceBytes, regexExpandBytes...)
-
-		// Replacement regex for binary files must account for null characters
-		replaceRegex = regexp.MustCompile(fmt.Sprintf(`%s([^\x00]*)`, quoteEscapeFind))
-		if replaceBytesLen > len(findBytes) {
-			logging.Errorf("Replacement text too long: %s, original text: %s", string(replaceBytes), string(findBytes))
-			return errors.New("replacement text cannot be longer than search text in a binary file")
-		} else if len(findBytes) > replaceBytesLen {
-			// Pad replacement with NUL bytes.
-			//logging.Debug("Padding replacement text by %d byte(s)", len(findBytes)-len(replaceBytes))
-			paddedReplaceBytes := make([]byte, len(findBytes)+len(regexExpandBytes))
-			copy(paddedReplaceBytes, replaceBytes)
-			replaceBytes = paddedReplaceBytes
-		}
-	} else {
-		replaceRegex = regexp.MustCompile(fmt.Sprintf(`%s`, quoteEscapeFind))
-		//logging.Debug("Assuming file '%s' is a text file", filename)
+	count, byts, err := replacePathInFile(fileBytes, find, replace)
```

Comment:
```suggestion
	count, byts, err := replaceInFile(fileBytes, find, replace)
```

Don't think this needs to have an opinion on what it is replacing.